code/mod_table/mod_table.cpp

```diff
@@ -538,6 +540,14 @@ void parse_mod_table(const char *filename)
 			}
 		}
 
+		if (optional_string("$Shockwaves Always Damage Bombs:")) {
+			stuff_boolean(&Shockwaves_always_damage_bombs);
+		}
+
+		if (optional_string("$Shockwaves Damage All Object Types Once")) {
```

Comment:
So close.  Missed the colon at the end. :)

Reply:
Not again!!!